src/components/Card/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+
+import "./Card.css";
+
+export interface ICardProps {
+  className?: string;
+  children?: React.ReactChildren | React.ReactNode | string;
+  responsive?: boolean;
+  responsiveColumns?: number;
+}
+
+const Card = (props: ICardProps) => {
```

Comment:
any reason this uses the function declaration instead of the class? I would prefer to use `class Card extends....`

Reply:
If the component doesn't have a state and too much logic, I usually write them using a function. Anyway, internally this function is mapped to a `React.Component`, so I can change the notation if you prefer it.

Reply:
I mean if there's no performance benefit, consistency is better. Otherwise we need to have guidelines on what not too much logic is for it to be a function, or when we add more logic here we might need to turn it back into a Class. Maybe I'm overthinking a little bit, but just some considerations.

Reply:
:+1: